src/client/temp/data/2016-ncnca-events.js

```diff
@@ -11970,18 +11970,18 @@ export default [{
   "discipline": "Cyclocross",
   "date": "October 08 2016",
   "location": {
-    "name": "Rocklin",
-    "streetAddress": "",
+    "name": "Johnson-Springview Park",
+    "streetAddress": "5480 5th St",
     "city": "Rocklin",
     "state": "CA",
-    "zip": "",
-    "lat": 38.780183,
-    "long": -121.230011
+    "zip": "95677",
+    "lat": "",
+    "long": ""
   },
   "resultsUrl": null,
   "promoters": [],
-  "promoterInfo": "http://www.saccyclocross.com/",
-  "registrationUrl": "",
+  "promoterInfo": "Official Event Website: http://www.saccyclocross.com/2016/Race1.html",
```

Comment:
You mean if it is a web site link, then I should create a new field "websiteUrl" and put it there? Since we don't have "websiteUrl" in our structure